faq/prooffacehelp.php

```diff
@@ -21,6 +21,10 @@
     echo "
     <center>
     <h2>Standard Proofreading Interface Help</h2>
+    <br>
+    <table>
+    <tr><td>Updated</td><td>05/24/2018</td><td>lhamilton</td></tr>
```

Comment:
Suggestion: Use ISO standard date format, 2018-05-24

(Though, I see we use US format below, so maybe consistency is best.)

Reply:
Yes, I'd prefer ISO-8601 as well but I think we should be consistent with the other dates in this file.